src/gg.regression.unity.experimental.runespawn/Editor/Runtime/SimulationInterface.cs

```diff
@@ -0,0 +1,133 @@
+    /// </code>
+    /// </example>
+    /// Example 3: Simulate a mouse click, drag and release:
+    /// <code>
+    /// MouseEventSender.SendMouseEvent(startTransform, leftButton: true);
+    /// yield return new WaitForSeconds(0.1f); // Wait for a few frames
+    /// MouseEventSender.SendMouseEvent(endTransform, leftButton: true);
+    /// yield return new WaitForSeconds(0.1f); // Wait for a few frames
+    /// MouseEventSender.SendMouseEvent(endTransform, leftButton: false);
+    /// </code>
+    public static void SendMouseEvent(Transform mouseTarget,
+                                      bool leftButton=false,
+                                      bool middleButton=false,
+                                      bool rightButton=false,
+                                      bool forwardButton=false,
+                                      bool backButton=false,
+                                      Vector2 scroll = default)
+    {
+        Vector2 screenPosition;
+        var boundsInfo = TransformObjectFinder.SelectBoundsForTransform(Camera.main, Screen.width, Screen.height, mouseTarget);
+        var ssBounds = boundsInfo.Item1;
+        if (ssBounds.HasValue)
+        {
+            screenPosition = ssBounds.Value.center;
+            SendMouseEvent(screenPosition, leftButton, middleButton, rightButton, forwardButton, backButton, scroll);
+        }
+        else
+        {
+            RGDebug.LogWarning($"mouseTarget transform is not on screen: {mouseTarget.name}");
+        }
+    }
+
+    /// <summary>
+    /// Simulates a mouse event at the specified screen position. Bottom left is (0,0) and top right is (Screen.width, Screen.height).
+    /// </summary>
+    /// <param name="mouseScreenPosition">The screen coordinates where the mouse event will be simulated.</param>
+    /// <param name="leftButton">Simulates left mouse button press if true.</param>
+    /// <param name="middleButton">Simulates middle mouse button press if true.</param>
+    /// <param name="rightButton">Simulates right mouse button press if true.</param>
+    /// <param name="forwardButton">Simulates forward mouse button press if true.</param>
+    /// <param name="backButton">Simulates back mouse button press if true.</param>
+    /// <param name="scroll">Simulates mouse scroll wheel movement.</param>
+    public static void SendMouseEvent(Vector2 mouseScreenPosition,
+                                      bool leftButton=false,
+                                      bool middleButton=false,
+                                      bool rightButton=false,
+                                      bool forwardButton=false,
+                                      bool backButton=false,
+                                      Vector2 scroll = default)
+    {
+        #if ENABLE_LEGACY_INPUT_MANAGER
+        MouseEventSender.SendMouseEventLegacy(mouseScreenPosition, Vector2.zero, scroll, leftButton, middleButton, rightButton, forwardButton, backButton);
```

Comment:
`SendRawPositionMouseEvent` already calls this. If you test this on a legacy input game, you will probably see double mouse events as currently coded

Reply:
@RG-nAmKcAz I tested it with both `Both and Input Manager (Old)`. Is this sufficient to switch? It sends the correct number clicks with that change in both.

Either way switched to only using `SendRawPositionMouseEvent`